support/ruby/collins-client/lib/collins/api/ip_address.rb

```diff
@@ -33,12 +33,12 @@ def ipaddress_update! asset_or_tag, old_address = nil, options = {}
       end
     end
 
-    def ipaddress_delete! asset_or_tag, pool = nil, address = nil
+    def ipaddress_delete! asset_or_tag, options = {}
       asset = get_asset_or_tag asset_or_tag
-      logger.debug("Deleting addresses for asset #{asset.tag} in pool #{pool}")
+      logger.debug("Deleting addresses for asset #{asset.tag} in pool #{options[:pool]} with address #{options[:address]}")
```

Comment:
will a missing key (say :address) print correctly or do you need to handle that better?